src/ConnectedMode.UnitTests/ConnectedMode.UnitTests.csproj

```diff
@@ -53,9 +53,9 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
     <PackageReference Include="MSTest.TestAdapter" Version="1.4.0" />
     <PackageReference Include="MSTest.TestFramework" Version="1.4.0" />
-
     <PackageReference Include="Moq" Version="4.18.2" />
     <PackageReference Include="FluentAssertions" Version="5.9.0" />
+    <Reference Include="PresentationFramework" />
```

Comment:
This package needed to be added for tests to build in 2019

Reply:
This is not a package tho. I don't understand it. Could you explain what is this in more detail?

Reply:
I'll put it in another ITemGroup reduce to confusion